test/extensions/tracers/datadog/agent_http_client_test.cc

```diff
@@ -28,12 +28,18 @@ namespace {
 
 struct InitializedMockClusterManager {
   InitializedMockClusterManager() {
+    using testing::DoAll;
```

Comment:
nit: usually we put these using decls at top level rather than repeating at call-sites.